processmanager/processinfo.go

```diff
@@ -37,6 +37,38 @@ import (
 	"go.opentelemetry.io/ebpf-profiler/util"
 )
 
+// isPIDLive checks if a PID belongs to a live process. It will never produce a false negative but
+// may produce a false positive (e.g. due to permissions) in which case an error will also be
+// returned.
+func isPIDLive(pid libpf.PID) (bool, error) {
```

Comment:
This is moved straight from the old `proc` package. But I wonder why it has two methods?
@athre0z any ideas why its kill + stat?
I suppose `kill` is faster, but perhaps does not work in certain containerized setups?